docs/getting-started/improvements.md

```diff
@@ -1 +1,19 @@
 # Improvements
+
+The following section contains suggestions for improving the application and may be extended much further.
+
+## Tiling of garden images to increase loading speed
+
+Administrators of a companies or gardens have the ability to upload a (self-taken) picture of a garden using
+the web interface. The image can be displayed in the dashboard-, beds- or plants-view in the web interface,
+replacing the satellite image at the exact location of the garden, providing a much more detailed
+view of the beds and plants.
+
+User images can have a large filesize, if taken with a high quality camera. This leads to long loading times
+of it in the web interface, especially for users having a slow internet connection. For the time downloading
+the image from the database, no image will be displayed.
```

Comment:
I don't understand the last sentence

Reply:
During the time that the image has not yet been loaded, it will not be displayed.